hq/app/utils/attempt/Failure.scala

```diff
@@ -64,6 +64,12 @@ object Failure {
     Failure(details, friendlyMessage, 500)
   }
 
+  def snykDashboardDecomission(cacheContent: String): Failure = {
+    val details = s"Cache service error; unable to retrieve $cacheContent"
+    val friendlyMessage = s"This dashboard has been temporarily closed due to snyk disabling the API it relied on. Please go to snyk.io directly for information about vulnerabilities in guardian services."
```

Comment:
This is true, although we could instead word it as `...closed while we evaluate the best format for this information`